README.md

```diff
@@ -1112,17 +1112,35 @@ dependencies {
 <details>
     <summary>Kapt</summary>
 
+##### Maven
+
 ```xml
 <dependency>
   <groupId>com.squareup.moshi</groupId>
   <artifactId>moshi-kotlin-codegen</artifactId>
-  <version>1.12.0</version>
+  <version>1.13.0</version>
   <scope>provided</scope>
 </dependency>
 ```
 
-```kotlin
-kapt("com.squareup.moshi:moshi-kotlin-codegen:1.13.0")
+##### Gradle
+
+In project's `build.gradle`
+```groovy
+plugins {
+  id "org.jetbrains.kotlin.kapt" version "1.7.10" // Or latest version of kapt
+}
+```
+
+In module's `build.gradle`
+```groovy
+plugins {
+  id 'kotlin-kapt'
+}
```

Comment:
Let's remove this, we're not trying to explain to users how to use kapt.

Reply:
done